<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TThermoFunMainWindow</class>
 <widget class="QMainWindow" name="TThermoFunMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome to Thermo Start Window</string>
  </property>
  <widget class="QWidget" name="centralWidget_">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Module</string>
    </property>
    <addaction name="actionThermo_Fun_Properties_at_TP"/>
    <addaction name="actionNew_Element_Window"/>
    <addaction name="actionNew_Substance_Window"/>
    <addaction name="actionNew_Reaction_Window"/>
    <addaction name="actionNew_Reaction_Se_t_Window"/>
    <addaction name="actionNew_Data_Source_Window"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Edge_Editor_Window"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionContents"/>
    <addaction name="separator"/>
    <addaction name="actionVersion"/>
    <addaction name="actionAuthors"/>
    <addaction name="actionLicense"/>
    <addaction name="separator"/>
    <addaction name="action_Help_About"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="action_Keep_Data_Fields_Expanded"/>
    <addaction name="action_Show_comments"/>
    <addaction name="action_Display_enums"/>
    <addaction name="action_Edit_id"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="upToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <property name="toolTip">
    <string>ThermoFun Thermodynamic data at T and P</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionThermo_Fun_Properties_at_TP"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="winToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string>Click desired icon to view data in the database</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionNew_Element_Window"/>
   <addaction name="actionNew_Substance_Window"/>
   <addaction name="actionNew_Reaction_Window"/>
   <addaction name="actionNew_Reaction_Se_t_Window"/>
   <addaction name="actionNew_Data_Source_Window"/>
   <addaction name="actionNew_ThermoDataSet_Window"/>
   <addaction name="actionNew_Edge_Editor_Window"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string>Click to set database connection and preferences</string>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="action_Help_About"/>
   <addaction name="actionPreferences"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/SetKeyTemplateIcon24.png</normaloff>:/Icons/SetKeyTemplateIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences ...</string>
   </property>
   <property name="iconText">
    <string>Preferences ...</string>
   </property>
   <property name="toolTip">
    <string>Preferences ...</string>
   </property>
  </action>
  <action name="action_Help_About">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ShowHelpWindowIcon24.png</normaloff>:/Icons/ShowHelpWindowIcon24.png</iconset>
   </property>
   <property name="text">
    <string> Help &amp;About</string>
   </property>
   <property name="toolTip">
    <string>Open Help Viewer ...</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Au&amp;thors ...</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>&amp;License ...</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Help &amp;Contents ...</string>
   </property>
  </action>
  <action name="action_Show_comments">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Display Comments to Data Fields</string>
   </property>
  </action>
  <action name="action_Display_enums">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Show Enum Names Instead of Values</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>&amp;Version ...</string>
   </property>
  </action>
  <action name="action_Edit_id">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/CancelIcon24.png</normaloff>:/Icons/CancelIcon24.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit _id of DB Documents (Records) </string>
   </property>
  </action>
  <action name="actionNew_Element_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ICompModuleIcon.png</normaloff>:/Icons/ICompModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Element Data (Vertex) ...</string>
   </property>
  </action>
  <action name="actionNew_Substance_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/DCompModuleIcon.png</normaloff>:/Icons/DCompModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Substance Data (Vertex) ...</string>
   </property>
  </action>
  <action name="actionNew_Reaction_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ReacDCModuleIcon.png</normaloff>:/Icons/ReacDCModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reaction Data (Vertex) ...</string>
   </property>
  </action>
  <action name="actionNew_Reaction_Se_t_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ReactionEquilibriumIcon.png</normaloff>:/Icons/ReactionEquilibriumIcon.png</iconset>
   </property>
   <property name="text">
    <string>ReactionSe&amp;t Data (Vertex) ...</string>
   </property>
  </action>
  <action name="actionNew_Data_Source_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/SDrefModuleIcon.png</normaloff>:/Icons/SDrefModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;DataSource (Vertex) ...</string>
   </property>
  </action>
  <action name="action_Keep_Data_Fields_Expanded">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Keep Data Fields Expanded</string>
   </property>
  </action>
  <action name="actionNew_Edge_Editor_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/DualThModuleIcon.png</normaloff>
     <normalon>:/main/GUI/Icons/DualThModuleIcon24.png</normalon>:/Icons/DualThModuleIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Graph Link Data (Edge) ...</string>
   </property>
  </action>
  <action name="actionThermo_Fun_Properties_at_TP">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/thermofun-logo-icon.png</normaloff>:/Icons/thermofun-logo-icon.png</iconset>
   </property>
   <property name="text">
    <string>Thermo&amp;Fun Properties at TP</string>
   </property>
  </action>
  <action name="actionRecord_Calculator">
   <property name="text">
    <string>Record &amp;Calculator</string>
   </property>
  </action>
  <action name="actionGenerate_a_ThermoDataSet_record">
   <property name="text">
    <string>Generate a ThermoDataSet record</string>
   </property>
  </action>
  <action name="actionNew_ThermoDataSet_Window">
   <property name="icon">
    <iconset resource="../ThermoFunDemoGUI.qrc">
     <normaloff>:/Icons/ThermoDatabasesIcon.png</normaloff>:/Icons/ThermoDatabasesIcon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;ThermoDataSet  (Vertex) ...</string>
   </property>
   <property name="toolTip">
    <string>ThermoDataSet (Vertex)</string>
   </property>
  </action>
  <action name="actionExport_ThermoDataSet">
   <property name="text">
    <string>Export ThermoDataSet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../ThermoFunDemoGUI.qrc"/>
 </resources>
 <connections/>
</ui>
